cmd/build/v2/build.go

```diff
@@ -167,9 +166,10 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 
 			buildSvcInfo := buildManifest[svcToBuild]
 
+			serviceContext := getServiceContext(buildSvcInfo)
```

Comment:
What is serviceContext? What is the purpose of it? Should we make the name of the function clearer?